examples/jetstream/04_ephemeral_consumer.ts

```diff
@@ -1,18 +1,18 @@
-import { connect } from "../src/nats_deno.ts";
-import { JetStreamManager } from "../src/jsm.ts";
+import { connect, consumerOpts } from "../../src/mod.ts";
 
 const nc = await connect();
 
-const jsm = await JetStreamManager(nc);
-const sub = await jsm.consumers.ephemeral("A", {}, {
-  manualAcks: false, // auto ack messages
-  max: 3,
-});
-const done = (async () => {
+const js = nc.jetstream();
+const opts = consumerOpts();
+opts.manualAck();
+opts.maxMessages(2);
+opts.deliverTo("xxx");
+const sub = await js.subscribe("a.>", opts);
```

Comment:
shouldnt this subscribe match deliverto?

Reply:
In this case the subscribe creates a consumer if not there, so it can only find the stream based on a subject mapped to the stream.